OpenRA.Mods.Common/ServerTraits/LobbyCommands.cs

```diff
@@ -766,11 +763,6 @@ static bool Kick(S server, Connection conn, Session.Client client, string s)
 				}
 
 				var kickClient = server.GetClient(kickConn);
-				if (server.State == ServerState.GameStarted && !kickClient.IsObserver)
```

Comment:
The server should still verify that the client being kicked is either an observer or a defeated player.